AI #967: Revise existing issue type templates and add a Feature Request issue type template (#955)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: .github/ISSUE_TEMPLATE/feature-request.md

- 5 = vague, high complexity or conceptual

```
[e.g., 2]
```

---

### 📂 Supporting Documentation
Include links to data samples, relevant PRs, GitHub discussions, or implementation references.

> 🔐 **Reminder:** Please ensure any linked documents are accessible to maintainers and collaborators. If access is restricted, your request may be delayed.

```
[Paste links here.]
```

---

### 🛠️ Proposed Solution / Approach
Share initial ideas, constraints, and feasibility considerations.

```
[Your proposal goes here.]
```

---

### 💬 Community Support (Add Your Voice)
If your organization supports this request or has a similar use case:

- Add a **comment** below including:
  - Your **organization**
  - A **brief explanation** of why this is important to you (e.g., use case, urgency)
- FOCUS Staff & Maintainers will aggregate supporting orgs over time.

_You can also submit support via [Google Form link] if you don’t have a GitHub account._

---

### 🔄 Maintainers Only
_To be filled in by the FOCUS Staff or Maintainers._

**Theme Association:**  
`[Optional grouping context]`

**Work Item ID:**  
`[Linked ID if applicable]`

**Status (select one):**

> - `Submitted` — newly filed, awaiting triage
> - `Needs More Info` — request needs clarification or detail before evaluation
> - `Under Consideration for Next Release` — being actively evaluated for upcoming spec work
> - `In Consideration for Future Releases` — valid idea, may be revisited later
> - `In Development` — linked to a work item and in progress
> - `Implemented` — published in a formal release of the spec
> - `Closed (Not Pursuing)` — out of scope, infeasible, or superseded

Reviewer: ```suggestion
- `Submitted` — newly filed, awaiting triage
- `Needs More Info` — request needs clarification or detail before evaluation
- `Under Consideration for Next Release` — being actively evaluated for upcoming spec work
- `Accepted in Scope` -- accepted for development in next release
- `In Development` — linked to a work item and in progress
- `Implemented` — published in a formal release of the spec
- `Closed (Not Pursuing)` — out of scope, infeasible, or superseded
```

Reviewer: I've now captured this in AI #862.  As agreed, we will strike this section and track as a status field.
